src/commands/towarticle.ts

```diff
@@ -0,0 +1,60 @@
+import { Command } from '../types';
+import fetch from 'node-fetch';
+import { polishPlurals } from 'polish-plurals';
+
+const MAX_RESULTS_NUMBER = 3;
+
+const pluralize = (count: number) => polishPlurals('artykuł', 'artykuły', 'artykułów', count);
+
+const typeofweb: Command = {
+  name: 'typeofweb',
+  description: 'Wyszukuje artykuły z Type of Web',
+  args: true,
+  async execute(msg, args) {
+    const query = encodeURIComponent(args.join(' '));
+    const res = await fetch(`https://typeofweb.com/wp-json/wp/v2/search?search=${query}`);
+
+    const data = (await res.json()) as ToWSearchResponse[];
+    if (!data[0].url.length) {
```

Comment:
To rzuci wyjątek, gdy nie zwrócono żadnych wyników.
```suggestion
    if (!data.length) {
```